app/components/Layout/Header.jsx

```diff
@@ -357,23 +356,7 @@ class Header extends React.Component {
             </a>
         );
 
-        let createAccountLink =
-            myAccountCount === 0 ? (
-                <ActionSheet.Button title="" setActiveState={() => {}}>
-                    <a
-                        className="button create-account"
-                        onClick={this._onNavigate.bind(this, "/create-account")}
-                        style={{padding: "1rem", border: "none"}}
-                    >
-                        <Icon
-                            className="icon-14px"
-                            name="user"
-                            title="icons.user.create_account"
-                        />{" "}
-                        <Translate content="header.create_account" />
-                    </a>
-                </ActionSheet.Button>
-            ) : null;
+        let createAccountLink = myAccountCount === 0 ? true : null;
```

Comment:
Can you elaborate why this was removed?

Reply:
Because createAccountLink used in 2 places only like a variable in conditions. It is not used like a React Element and is not displayed anywhere. I've checked it twice.